presto-docs/src/main/sphinx/connector/oracle.rst

```diff
@@ -23,47 +20,350 @@ connection properties as appropriate for your setup:
+    presto> INSERT INTO vals (col) VALUES (CAST('BB' as CHAR(2001)));
+    presto> SELECT LENGTH(col) FROM vals;
+    2001
+    1
+
+Attempting to write a ``CHAR`` that doesn't fit in the column's actual size
+fails. This is also true for the equivalent ``VARCHAR`` types.
+
+Type mapping configuration properties
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. list-table:: Type Mapping Properties
+  :widths: 20, 20, 50, 10
+  :header-rows: 1
+
+  * - Configuration property name
+    - Session property name
+    - Description
+    - Default
+  * - ``unsupported-type.handling-strategy``
+    - ``unsupported_type_handling_strategy``
+    - Configures how unsupported column data types are handled:
+
+      - ``IGNORE`` - column is not accessible.
+      - ``CONVERT_TO_VARCHAR`` - column is converted to unbounded ``VARCHAR``.
+
+    - ``IGNORE``
+  * - ``oracle.number.default-scale``
+    - ``number_default_scale``               
+    - Default Presto ``DECIMAL`` scale for Oracle ``NUMBER`` (without precision 
+      and scale) date type. When not set then such column is treated as not 
+      supported.
+    - not set
+  * - ``oracle.number.rounding-mode``
+    - ``number_rounding_mode``
+    - Rounding mode for the Oracle ``NUMBER`` data type. This is useful when 
+      Oracle ``NUMBER`` data type specifies higher scale than is supported in 
+      Presto. Possible values are:
+
+      - ``UNNECESSARY`` - Rounding mode to assert that the 
+        requested operation has an exact result, 
+        hence no rounding is necessary.
+      - ``CEILING`` - Rounding mode to round towards
+        positive infinity.
+      - ``FLOOR`` - Rounding mode to round towards negative
+        infinity.
+      - ``HALF_DOWN`` - Rounding mode to round towards
+        ``nearest neighbor`` unless both neighbors are
+        equidistant, in which case rounding down is used.
+      - ``HALF_EVEN`` - Rounding mode to round towards the
+        ``nearest neighbor`` unless both neighbors are equidistant,
+        in which case rounding towards the even neighbor is
+        performed.
+      - ``HALF_UP`` - Rounding mode to round towards
+        ``nearest neighbor`` unless both neighbors are
+        equidistant, in which case rounding up is used
+      - ``UP`` - Rounding mode to round towards zero.
+      - ``DOWN`` - Rounding mode to round towards zero.
+
+    - ``UNNECESSARY``
+
+Synonyms
+--------
+
+Based on performance reasons, Presto disables support for Oracle ``SYNONYM``. To
+include ``SYNONYM``, add the following configuration property: 
+
+.. code-block:: none
+
+    oracle.synonyms.enabled=true
+
+Pushdown
+--------
+
+The connector supports :doc:`pushdown </optimizer/pushdown>` for optimized query processing.
 
 Limitations
 -----------
 
-The following SQL statements are not yet supported:
+The following SQL statements are not supported:
 
 * :doc:`/sql/delete`
+* :doc:`/sql/alter-table`
+* :doc:`/sql/create-table` (:doc:`/sql/create-table-as` is supported)
```

Comment:
@losipiuk can you confirm that `Create table` isn't supported and that `create tables as` is supported? David asked me to double check. Thanks!

Reply:
Just checked. Create table is also supported.
@findepi do we have a test coverage for that? Quickly I could not find any.

Something as simple as
```java
    @Test
    public void testCreateTable()
    {
        getQueryRunner().execute("CREATE TABLE t (c BIGINT)");
        getQueryRunner().execute("INSERT INTO t VALUES(10)");
        assertThat(query("SELECT CAST(c as VARCHAR) from t")).matches("VALUES (VARCHAR '10')");
    }
```

or 

```java
    @Test
    public void testCreateTable()
    {
        getQueryRunner().execute("CREATE TABLE t (c BIGINT)");
        assertThat(query("DESCRIBE t")).matches(results -> results.getRowCount() > 0);
    }
```

Reply:
`io.prestosql.testing.AbstractTestDistributedQueries#testCreateTable`

Reply:
@losipiuk thanks so much! I'm going to create a PR to remove `create tables` and the note about `create tables as` from the limitations section.

Reply:
> io.prestosql.testing.AbstractTestDistributedQueries#testCreateTable

Cool. I was looking in the `*Smoke*` ones.